imports/region/Region.js

```diff
@@ -231,159 +224,129 @@ class Region extends Component {
             console.log('drag rect:', x, ';', y);
             this.moveRect(x, y, index);
           }}
-          // onDragEnd={(e) => {
-          //   console.log(`dimensions: (${e.target.attrs.x}, ${e.target.attrs.y}),
-          //   (${e.target.attrs.x + e.target.attrs.width}, ${e.target.attrs.y}),
-          //   (${e.target.attrs.x}, ${e.target.attrs.y + e.target.attrs.height}),
-          //   (${e.target.attrs.x + e.target.attrs.width}, ${e.target.attrs.y + e.target.attrs.height})`);
-          // }}
+
           onClick={() => {
             this.setState({
-              toDelete: item.key,
+              selectedRegionKey: region.key,
             });
           }}
-          // ref={(node) => {
-          //   if (node && !this.regions[item.key].hasOwnProperty('shape')) {
-          //     this.regions[item.key].shape = node;
-          //     this.regions[item.key].shape.on('dragmove', () => {
-          //       // console.log('dragmove');
-          //       // const itemW = item.w;
-          //       // const itemH = item.h;
-          //       // const i = index;
-          //       // this.props.regionArray.forEach((obj, index) => {
-          //       //   if (obj.key === item.key) {
-          //       //     itemW = obj.w;
-          //       //     itemH = obj.h;
-          //       //     i = index;
-          //       //   }
-          //       // });
-          //       const x = this.regions[item.key].shape.getAttrs().x;
-          //       const y = this.regions[item.key].shape.getAttrs().y;
-          //       this.moveRect(x, y, index);
-          //       // this.reshape(itemW, itemH, x, y, i);
-          //     });
-          //     this.regions[item.key].shape.on('click', () => {
-          //       this.setState({
-          //         toDelete: item.key,
-          //       });
-          //     });
-          //   }
-          // }}
+
         />
         {anchors}
       </Group>
     );
     return result;
-  }
-  handleTouchTap = (event) => {
+  };
+
+  handleTouchTap = (event: SyntheticTouchEvent<HTMLButtonElement>) => {
```

Comment:
ref: https://flow.org/en/docs/react/events/
I don't think we can get much benefit from this **long** writing since it does not really show the internal data structure of one event. My suggestions is, not `type here`.

Reply:
Yes, if we're purely interested in the target, there's no need for a type here.